src/analysis/normed_space/add_torsor.lean

```diff
@@ -226,10 +226,10 @@ calc edist (f x -ᵥ g x) (f y -ᵥ g y) ≤ edist (f x) (f y) + edist (g x) (g
   (add_mul _ _ _).symm
 
 lemma uniform_continuous_vadd : uniform_continuous (λ x : V × P, x.1 +ᵥ x.2) :=
-(lipschitz_with.prod_fst.vadd lipschitz_with.prod_snd).uniform_continuous
+((@lipschitz_with.prod_fst V P _ _).vadd lipschitz_with.prod_snd).uniform_continuous
```

Comment:
Hmm, when this problem appeared in #6746 I thought it was because of the product of `semi_normed_group`, but I see now that they come from the generalization of Lipschitz maps to `pseudo_emetric_space`. Once we will have `semi_normed_space` I will try to generalize this whole file to see if the original version works, without the `@`.

Reply:
Sometimes Lean fails to use dot notation on not fully elaborated terms, sometimes it doesn't. I don't know why it happens.